src/rdiff-backup-delete

```diff
@@ -25,13 +25,17 @@
 #
 
 import os
```

Comment:
Overall, the imports should be in alphabetical order.

Reply:
https://www.python.org/dev/peps/pep-0008/#imports doesn't say anything about it, I have a slight preference for the 2nd approach, which is also what you chose, so fine with me.